jumpscale/clients/gedis/gedis.py

```diff
@@ -83,15 +86,36 @@ class GedisClient(Client):
     hostname = fields.String(default="localhost")
     port = fields.Integer(default=16000)
     raise_on_error = fields.Boolean(default=False)
+    server_tid = fields.String()
 
     def __init__(self):
         super().__init__()
         self._redisclient = None
         self._loaded_actors = {}
         self._loaded_modules = []
         self.actors = None
+        self.explorer = j.clients.explorer.get_default()
+        self.identity = j.core.identity
+        self._auth_server()
         self._load_actors()
 
+    def _get_user_pubkey(self):
+        server_tid = self.server_tid or self.identity.tid
```

Comment:
this looks strange why would it be the identity of the user?

Reply:
By default the client connects to a local server so they would be the same user

Reply:
wouldn't it make more sense to set self.server_tid to self.iodentity,tid in the contructor for such a case instead of this logic here?